src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css

```diff
@@ -119,38 +119,75 @@
     height: 1.2em !important;
 }
 
-@media all and (max-width: 87.5em) {
+@media all and (max-width: 73em) {
 
-    .nowPlayingBarUserDataButtons {
+    .nowPlayingBarRight .nowPlayingBarUserDataButtons {
         display: none;
     }
 }
 
-@media all and (max-width: 68.75em) {
+@media all and (max-width: 70em) {
 
-    .nowPlayingBarVolumeSliderContainer, .nowPlayingBar .muteButton, .nowPlayingBar .unmuteButton {
+    .nowPlayingBarRight .toggleRepeatButton {
+        display: none;
+    }
+}
+
+@media all and (max-width: 67em) {
+
+    .nowPlayingBarCenter .nowPlayingBarCurrentTime {
         display: none !important;
     }
+
 }
 
-@media all and (max-width: 50em) {
+@media all and (max-width: 61em) {
 
-    .nowPlayingBarCenter {
+    .nowPlayingBarCenter .stopButton {
```

Comment:
Why did you add the stopButton class here? this is inside the div with the class nowPlayingBarCenter

Reply:
also forget this one, you are only applying the rule to stopButton, makes sense.